examples/bookstack.md

```diff
@@ -0,0 +1,60 @@
+# Bookstack (non-docker version)
+
+## On Kanidm 
+### 1. Create the bookstack resource server
+```
+kanidm system oauth2 create bookstack "Bookstack" https://yourbookstack.example.com
+```
+### 2. Create the appropriate group(s)
+```
+kanidm group create bookstack-users --name idm_admin
+```
+### 3. Add the appropriate users to the group
+```
+kanidm group add-members bookstack-users user.name
+```
+### 4. Add the scopes:
+```
+kanidm system ouath2 update-scope-map bookstack openid profile email keys
+```
+### 5. Get the client secret:
+```
+kanidm system oauth2 show-basic-secret bookstack
+```
+Copy the value that is returned.
+
+### 6. Disable PKCE / Enable Legacy crypto
+```
+kanidm system oauth2 warning-insecure-client-disable-pkce bookstack
```

Comment:
It is not necessary to disable PKCE anymore. See https://github.com/BookStackApp/BookStack/issues/4734